migration/src/m20230622_104142_stripe_deposits.rs

```diff
@@ -0,0 +1,104 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .create_table(
+                Table::create()
+                    .table(StripeIncreaseBalanceReceipt::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(StripeIncreaseBalanceReceipt::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(
+                        ColumnDef::new(StripeIncreaseBalanceReceipt::StripePaymentIntendId)
+                            .string()
+                            .not_null(),
+                    )
+                    .col(
+                        // I will not mark this as "not null", so we can track who to refund more easily if needed
+                        ColumnDef::new(StripeIncreaseBalanceReceipt::DepositToUserId)
+                            .big_unsigned(),
```

Comment:
Unsigned

Reply:
this should be big-unsigned, no? otherwise it complains on my end about wrong foreign keys types